### ISSUE TITLE
Support `Panic` keyword in catch block error types of try/catch

### ISSUE THREAD

**Issue by Amxx** [sc-forks/solidity-coverage#608]
I am getting 
```
Error in plugin solidity-coverage: Error: Could not instrument: xxx/xxx.sol. (Please verify solc can compile this file without errors.) Expected "Error" identifier in catch clause
```
with a contract that uses 0.8.1 mechanism to catch Panic

**Comment by cgewecke** [comment]
@Amxx 

Could you show the complete stack trace of the error and a slice of the Solidity code which throws this?

**Comment by cgewecke** [comment]
[solidity-parser/parser 39][1]

Example from tests in that PR
```solidity
try f(1, 2) returns (uint a) {
  // try
} catch Panic(uint errorCode) {
  // catch panic
} catch (bytes memory c) {
  // catch low level err
}
```

[1]: https://github.com/solidity-parser/parser/pull/39

**Comment by fvictorio** [comment]
Sorry about this. I will publish a new version of the parser today.

**Comment by cgewecke** [comment]
@fvictorio Oh no worries - had no idea what this was about until I saw your tests 😅

**Comment by fvictorio** [comment]
Fixed in version `0.12.0` of the parser!

**Comment by cgewecke** [comment]
Should be fixed in `0.7.16`.  Just lmk if you're still running into any problems with solc 0.8.x.
